### PR TITLE
Merge token transfer and native token transfer

### LINKED ISSUE #42: Merge token transfer and native token transfer

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #42

### CODE REVIEW

File: types/tx.go [hunk 1]
```diff
@@ -21,8 +21,7 @@ const (
 	DirectionSelf     Direction = "yourself"
 
 	TxTransfer              TransactionType = "transfer"
-	TxNativeTokenTransfer   TransactionType = "native_token_transfer"
```
Comment (vikmeup) [inline types/tx.go:24]: You need to keep native token transfer here - all toke transfers will use it

**Review by vikmeup — APPROVED** [review]

File: types/tx.go (hunk 1 above)
Comment (hewigovens) [reply types/tx.go:24]: just use `TxTokenTransfer`?

Comment (hewigovens) [reply types/tx.go:24]: `TxTokenTransfer` is `native_token_transfer` now

Comment (vikmeup) [reply types/tx.go:24]: 👍

**Review by vikmeup — APPROVED** [review]

File: types/tx.go (hunk 1 above)
Comment (hewigovens) [reply types/tx.go:24]: after checking iOS code, I guess we'd better to keep both `TxNativeTokenTransfer` and `TxTokenTransfer` here? so it won't break its logic

Comment (vikmeup) [reply types/tx.go:24]: We can only return native transfers for all. For consistency

Comment (vikmeup) [reply types/tx.go:24]: Or wait until new V2 format is ready - that works too

**Comment by hewigovens** [comment]
@ivan-bozhytksyi could you please handle this in v2?

**Comment by ivan-bozhytksyi** [comment]
@hewigovens yes, I just wrote about it in the related ticket. You can close this PR. 
BTW, we have moved new tx type to v1 package, so let's call it v1 :)

**Pull request closed without merging**